vela-templates/definitions/internal/workflowstep/notification.cue

```diff
@@ -201,10 +201,20 @@ template: {
 	}
 
 	option: {
-		text:         textType
-		value:        string
-		description?: textType
-		url?:         string
+		text: {
+			type:      string
+			text:      string
+			emoji?:    bool
+			verbatim?: bool
+		}
+		value: string
+		description?: {
+			type:      string
+			text:      string
+			emoji?:    bool
+			verbatim?: bool
+		}
+		url?: string
```

Comment:
why don't use the textType here?

Reply:
It will cause cue panic here, refer to issue: cue-lang/cue#1829